src/pages/Controllers/Controllers.js

```diff
@@ -37,19 +40,34 @@ export default function Controllers() {
         { content: "-", className: "u-align--right" },
         { content: "-", className: "u-align--right" },
         { content: "-", className: "u-align--right" },
-        { content: `${c.Public}`, className: "u-align--right" }
+        { content: `${c.Public}`, className: "u-align--right u-capitalise" }
       ]
     }));
 
   return (
     <Layout>
       <div className="l-content">
+        <div className="overview">
+          <div className="row">
+            <div className="col-4 overview__machines">
```

Comment:
I'm not a fan of using class names to target elements in for tests. I believe class names should be reserved for styling. Is there another way to do this? Maybe a data attr for example: `data-test="overview__machines"`.

Reply:
I'd prefer this approach also, although think we should use `data-testid` to be explicit. 

Note, however, those who have given this much more thought [0], recommend that targeting via a data attribute is used as a last resort. [1]

However, I like the simplicity of using it consistently everywhere.  

[0] https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change
[1] https://testing-library.com/docs/guide-which-query

Reply:
I agree, while I'm not against using classes for targeting if you have already added a specific class to style the target, in this case there isn't really a clean way of accessing it. I came up with something like...

```js
  it("displays the correct counts", () => {
    const store = mockStore(dataDump);
    const wrapper = mount(
      <MemoryRouter>
        <Provider store={store}>
          <Controllers />
        </Provider>
      </MemoryRouter>
    );
    const values = [
      "56 machines",
      "59 applications",
      "58 units"
    ];
    wrapper.find('.overview strong').forEach((ele, index) => expect(ele.text()).toBe(values[index]));
  });
```

Which isn't super resilient to markup changes, which is kind of the whole point of not using classes to query for elements :-D But maybe it's ok in this case...however, maybe in this case using `data-testid` (or whatever string we'd like to standardize on) is more appropriate.